healthcheck/tools/db/db.go

```diff
@@ -36,6 +36,10 @@ func Dial(conf *Config) (*mgo.Client, error) {
 		"ssl_secure": conf.SSL.Insecure,
 	}).Debug("Connecting to mongodb")
 
+	if err := conf.configureTLS(); err != nil {
```

Comment:
@tplavcic @egegunes please add this conf to one of our test 
```
    configuration: |
      net:
        tls:
          mode: requireTLS
```

Reply:
As discussed we will add a test change in a separate PR. This looks OK for me.